torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -2079,7 +2087,8 @@ compute_bucket_assignment_by_size(
 // across processes.
 void verify_replica0_across_processes(
     c10::intrusive_ptr<c10d::ProcessGroup> process_group,
-    std::vector<std::vector<at::Tensor>> model_replicas) {
+    std::vector<std::vector<at::Tensor>> model_replicas,
```

Comment:
nit: pass by const reference? Since each tensor has metadata that won't be shared automatically like storage, still better to use reference here.